src/main/java/frc/robot/Other/Explanations.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.Other;
+
+import frc.robot.subsystems.*;
+import frc.robot.Constants;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+import frc.robot.commands.*;
+
+/**
+ * Here, Explanations of How everything works are held.
+ *
+ * <p>When Within The class:
+ *
+ * <p>{@code - Hover over the Green Part to Get a basic explanation}
+ *
+ * <p>{@code - Hover over the Yellow Part for a More Detailed explanation}
+ *
+ * <p>{@code - Hover over The Blue Part to find what your getting an Explanation of}
+ *
+ * <p>EX: int {@link #DriveTrainExplanation} (Blue) = {@link DriveTrain} (Green) {@link
+ * DriveTrain#Explanation} (Yellow)
+ *
+ * <p>Sub Explanations are down Below:
+ *
+ * <ul>
+ *   <li>Init Sendable:
+ *       <ul>
+ *         <li>Sendable objects are things that can be sent to the smart dashboard (WHEN TALKING
+ *             ABOUT FRC!!!!).
+ *         <li>Initializing a sendable object sends that object to the smart dashboard.
+ *         <li>To send other things to the smart dashboard, Use the sendable builder (the thing that
+ *             sends stuff)
+ *         <li>EX: {@link Hanger#initSendable} and {@link DriveTrain#initSendable}
+ *         <li>In order to send variables, you must do {@code builder.addDoubleProperty} (Instead of
+ *             Double do variable type of choice)
+ *       </ul>
+ */
```

Comment:
What are these HTML properties for?

Reply:
If you go Into Vs code you  will see. Basically how to add bullet points and other cool stuff like that. Some HTML features work with JAVA DOC

Reply:
Got it. Makes sense.